Gemfile.lock

```diff
@@ -1274,8 +1266,6 @@ DEPENDENCIES
   shoulda-matchers
   shrine
   sidekiq (~> 7.2.0)
-  sidekiq-ent!
-  sidekiq-pro!
```

Comment:
I don't think sidekiq things should be removed.

Reply:
Have you set the enterprise license key locally? https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/platform/engineering/sidekiq-enterprise-setup.md

Reply:
I have not.  Thanks for the link.